docs/performance-guide.md

```diff
@@ -25,150 +54,248 @@ guide intends to answer the following questions:
 
 -   What is the status of ASRS performance for each tier?
 
--   Does ASRS meet my requirement to send 100,000 message per second?
+-   Does ASRS meet my requirement of message rate, for example, send for 100,000 message per second?
 
--   For my specific scenario, which tier is suitable for me?
+-   For my specific scenario, which tier is suitable for me? Or how can I select the proper tier?
 
 -   What kind of app server (VM size) is suitable for me and how many of them shall I deploy?
 
+To answer those questions, the performance guide first gives a high level explanation about the factors which take impact on performance, then illustrates the maximum inbound
+and outbound message for every tier through 4 typical use cases: **echo**, **broadcast**,
+**send to group**, and **send to connection** (peer to peer chatting).
 
-To answer those questions, the performance guide illustrates the max inbound
-and outbound message for every tier on 4 typical use cases: echo, broadcast,
-send to group, and send to connection (peer to peer chatting).
+Different user has different use cases. It is impossible for this document to enumerate all scenarios (different use case, different message size, or message sending
+pattern etc.) in the limited paragraph, however, it provides some evaluation methods to help users to approximately evaluate their requirement of the inbound or outbound message, then find the proper tiers by checking the performance table. 
 
-Different user has different use cases. It is impossible for this document to enumerate all
-scenarios (different use case, different message size, or message sending
-pattern etc.) in the limited paragraph, however, it provides a formula for users to approximately
-evaluate their specific max inbound or outbound based on the 4 known use cases.
+<a name="perf_insight"></a>
+Performance insight
+-------------------
 
-Guide
------
+This section first describes the performance evaluation methodologies, then lists all factors that take impact on performance, at last provides methods to help evaluate the performance requirements.
+
+<a name="perf_methodology"></a>
+### Methodology
+
+**Throughput** and **latency** are two typical aspects of performance checking. For ASRS, different SKU tier has different throughput throttling policy. This document wants to find **the maximum allowed throughput (inbound and outbound bandwidth) with 99% of messages latency less than 1 second**.
```

Comment:
Describe what's the definition of latency here.

Reply:
Done